robottelo/decorators/host.py

```diff
@@ -0,0 +1,120 @@
+import logging
+from functools import wraps
+
+import re
+import six
+import unittest2
+
+from robottelo import ssh
+from robottelo.config import settings
+
+if six.PY3:
+    from functools import lru_cache
+elif six.PY2:
+    # workaround only for this module, its not doing exactly the same as
+    # original from Python 3, but it fits this specific case
```

Comment:
OK, sorry I misunderstood the code.. thanks.

Maybe we can put this in robottelo.helpers and here just `from robottelo.helpers import lru_cache` ?